marketplace-image.json

```diff
@@ -1,10 +1,9 @@
 {
   "variables": {
     "do_token": "{{env `DIGITALOCEAN_TOKEN`}}",
-    "image_name": "lemp-20-04-snapshot-{{timestamp}}",
-    "apt_packages": "fail2ban mysql-server nginx php-apcu php-curl php-mysql php-fpm postfix python3-certbot-nginx software-properties-common",
-    "application_name": "LEMP",
-    "application_version": ""
+    "image_name": "osmosis-20-04-{{user `application_version`}}-{{timestamp}}",
```

Comment:
is the `{{timestamp}}` needed?

Should we just call the image with the `application_version` ?
wdyt?

Reply:
I personally like having the timestamp, especially when I'm testing things and will have multiple images called "osmosis-20-04-7.2.0" and dont want them overrided. This also wont be user facing, these image names will only be seen by DO marketplace team.